homeassistant/components/utility_meter/config_flow.py

```diff
@@ -118,7 +127,7 @@ async def _validate_config(
 class ConfigFlowHandler(SchemaConfigFlowHandler, domain=DOMAIN):
     """Handle a config or options flow for Utility Meter."""
 
-    VERSION = 2
+    VERSION = 3
```

Comment:
When we use optional for this new option, we don't need to change the version and therefore we don't need a migration.

Reply:
You're right... I removed that.